app/src/main/java/com/taskruler/MainActivity.kt

```diff
@@ -32,13 +32,13 @@ class MainActivity : ComponentActivity() {
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
                 ) {
-                    Greeting("Android")
+                    LogActivity("Android")
```

Comment:
“Greeting” is not longer the best description of the function creating the UI for the home page/initial page you see when launching the app. If not my suggestion (LogActivity) maybe Home, LogScreen, etc.

Reply:
Agreed.  Make function names self-describing.